src/App.tsx

```diff
@@ -1,96 +1,57 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import { getNumbers } from './utils';
+import { getEdgeIndexes, getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import { Selector } from './components/Selector';
+import { ItemsList } from './components/ItemsList/ItemsList';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
+const TOTAL_PAGES = 42;
+const VALUES = [3, 5, 10, 20];
```

Comment:
use enum here
change variable name to something like perPage / perPageValues / perPageOptions